Server/Services/CommandService.cs

```diff
@@ -0,0 +1,65 @@
+﻿using IgiCore.Core.Services;
+using System.Collections.Generic;
+using Citizen = CitizenFX.Core.Player;
+using IgiCore.Core.Commands;
+using System.Linq;
+using System.Threading.Tasks;
+using IgiCore.Server.Commands;
+
+namespace IgiCore.Server.Services
+{
+	public class CommandService : Service, IService
```

Comment:
I think this would be better suited as a "Manager" rather than "Service"?

Reply:
I did a service as it handles commands, and automatically registers them when it starts like the other services, although it could be a manager as well.